### PR TITLE
Add "Get issue description" feature to our Telegram bot

### PR DESCRIPTION
**Pull request by VitalinaMykoliuk**

### CODE REVIEW

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Remove `Scriptrs` folder, you shouldn't commit it to repository

File: pyvenv.cfg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+home = C:\Users\HP\AppData\Local\Programs\Python\Python311
```
Comment (djeck1432) [inline pyvenv.cfg:1]: Remove this file, it should be on your local machine but not in repository

Comment (iamnovichek) [inline pyvenv.cfg:1]: Update .gitignore file to not detect this extension.

File: tracker/telegram/bot.py [hunk 2]
```diff
@@ -113,6 +113,17 @@ async def send_deprecated_issue_assignees(msg: Message) -> None:
         await msg.reply(f"<blockquote>{message}</blockquote>")
 
 
+def escape_html(text: str) -> str:
+    """
+    Escapes HTML symbols in the text to ensure proper rendering in Telegram messages.
```
Comment (iamnovichek) [inline tracker/telegram/bot.py:118]: Please add also :params: and :return: in your docstring

**Review by iamnovichek — CHANGES_REQUESTED** [review]

**Pull request merged**